lxd/apparmor/instance_qemu.go

```diff
@@ -83,6 +83,12 @@ profile "{{ .name }}" flags=(attach_disconnected,mediate_deleted) {
 
   # Snap-specific libraries
   /snap/lxd/*/lib/**.so*            mr,
+
+{{- if .extQemuSnap }}
+  /snap/lxd/*/opt/external-qemu/lib/**.so*        mr,
+  /snap/lxd/*/opt/external-qemu/bin/qemu-system-* mrix,
```

Comment:
Would that also need rules to allow reading potential ROMs and keymaps?

Reply:
IMHO, any additional things can be covered by per-instance `raw.qemu` option if needed. As this external qemu thing is a bit unusual, it's hard to predict what it can require in practice. I don't think that we can provide any kind of universal AppArmor profile here and at the same time safe. But if you have any specific rule in mind - let's add it.

Reply:
I had `/snap/lxd/*/opt/external-qemu/share/** r,` in mind.

Reply:
Thanks! I have added this.